src/webpack.dev.config.js

```diff
@@ -30,16 +30,18 @@ const config = deepAssign({
 }, options.webpack);
 
 // Ensure dev loaders are used.
-config.module.loaders = config.module.loaders.map(loader => {
-  if (loader.test.test('.jsx')) {
-    loader.loader = `react-hot!babel`;
-  } else if (loader.test.test('.scss')) {
+config.module.loaders = config.module.loaders.map(entry => {
+  let {loader} = entry;
+  if (/babel/.test(loader)) {
+    loader = `react-hot!babel`;
```

Comment:
sorry for not catching it earlier. but it is possible to chain babel loaders as well. can we use replace in a similar way we did with scss loaders?

loader = loader.replace(/babel/, 'react-hot!$1')